### PR TITLE
Fix: Fixed issue where progress was not updated when creating an archive 

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers..

**Archive Name**
- Operation banner was not reporting the name of the archive being created. This PR fixes that issue

**Archive Progress**
- This PR doesn't completely resolve the issue (I'll try to open a different PR in SevenZip repo to fix that). 
- If there are at least two folders, progress will be shown correctly (4 folders: 0% -> 25% -> 50% -> 75% -> Completed)

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ x Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Select at least two folders
   2. Create an archive
   3. See that progress is now being reported

### CODE REVIEW

File: src/Files.App/Filesystem/Archive/ArchiveCreator.cs [hunk 1]
```diff
@@ -11,8 +11,15 @@ namespace Files.App.Filesystem.Archive
 {
 	public class ArchiveCreator : IArchiveCreator
 	{
+		private int itemsAmount = 1;
```
Comment (yaira2) [inline src/Files.App/Filesystem/Archive/ArchiveCreator.cs:14]: What's this for?

Comment (ferrariofilippo) [reply src/Files.App/Filesystem/Archive/ArchiveCreator.cs:14]: It's needed to calculate a weighted percentage of progress. If you compress multiple folders, each one will report its own progress. Using `itemsAmount` we can report a realistic progress.
To get the actual progress, I use this formula: 

    [oldPercentage + (percentageDelta / itemsAmount)] 

which means that if we are compressing 2 items and we have compressed 50% of the first one, the total progress will be:
   
    [0% + (50% / 2)] = 25%

Comment (yaira2) [reply src/Files.App/Filesystem/Archive/ArchiveCreator.cs:14]: Can you add a comment to the code?

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**